GetIntoTeachingApi/Controllers/TeacherTrainingAdviser/CandidatesController.cs

```diff
@@ -46,7 +45,7 @@ ICrmService crm
                 return BadRequest(this.ModelState);
             }
 
-            return Ok(new Object());
+            return Ok(new object());

```

Comment:
conventionally caps, resharper is wrong

Reply:
ill not comment on each instance of Object

Reply:
These are temporary anyway - they'll be updated to specific objects in future PRs